### PR TITLE
docs(*) add Universal setup to README.md

### PR DESCRIPTION
**Pull request by tomaszwylezek**
Signed-off-by: Tomasz Wylężek <tomwylezek@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,20 +27,93 @@ The `zone` key is purely static and arbitrary, but by having different `zone` va
...
     $ redis-server
     $ redis-cli set zone local
     ```
 
-1.  Start `demo-app` on the default port `5000`:
+    - (Universal setup) on port `26379` and set a default `zone` name:
+
+     ```sh
+    $ redis-server --port 26379
+    $ redis-cli set zone local
```
Comment (lahabana) [inline README.md:43]: Seems like the port needs to be set: ` redis-cli -p 26379 set zone local`

File: README.md [hunk 2]
```diff
@@ -27,20 +27,93 @@ The `zone` key is purely static and arbitrary, but by having different `zone` va
...
+    ```
+
+2.  Start `demo-app` on the default port `5000`:
 
     ```sh
     $ npm start --prefix=app/
```
Comment (lahabana) [inline README.md:49]: On a fresh checkout I needed to do `npm install --prefix=app/`

Comment (lahabana) [reply README.md:49]: Also the app by default is trying to connect to redis on 6379 which won't work until the dp is started

File: README.md [hunk 3]
```diff
@@ -27,20 +27,93 @@ The `zone` key is purely static and arbitrary, but by having different `zone` va
...
     ```
 
-1.  Navigate to [`127.0.0.1:5000`](http://127.0.0.1:5000) and increment the counter!
+3.  Navigate to [`127.0.0.1:5000`](http://127.0.0.1:5000) and increment the counter!
```
Comment (lahabana) [inline README.md:52]: It might be worth saying that if the DPs are not started it won't work?

File: README.md [hunk 4]
```diff
@@ -27,20 +27,93 @@ The `zone` key is purely static and arbitrary, but by having different `zone` va
...
+
+
+### Run in Universal
+
+1. First we need to genarate two tokens:
```
Comment (lahabana) [inline README.md:57]: nit: `s/genarate/generate`

File: README.md [hunk 5]
```diff
@@ -27,20 +27,93 @@ The `zone` key is purely static and arbitrary, but by having different `zone` va
...
 
     ```sh
     $ npm start --prefix=app/
     ```
 
-1.  Navigate to [`127.0.0.1:5000`](http://127.0.0.1:5000) and increment the counter!
+3.  Navigate to [`127.0.0.1:5000`](http://127.0.0.1:5000) and increment the counter!
+
+
+### Run in Universal
```
Comment (lahabana) [inline README.md:55]: Should there be a very first step to get a cp running and kumactl installed (or at least pointing to the docs on how to do that?

**Review by lahabana — COMMENTED** [review]
I've tried following the steps on a fresh checkout.

File: README.md (hunk 3 above)
Comment (tomaszwylezek) [reply README.md:52]: I think we might think about removing this step? Or add info that it only works in Kubernetes steps.

File: README.md (hunk 4 above)
Comment (tomaszwylezek) [reply README.md:57]: nice catch!

File: README.md (hunk 2 above)
Comment (tomaszwylezek) [reply README.md:49]: You are right, I will add that step

File: README.md (hunk 5 above)
Comment (tomaszwylezek) [reply README.md:55]: We might think about it later on but for now, I would leave it as it is (As I only wanted to add information about how to start with Universal mode).

**Pull request merged**